pkg/api/pod/util.go

```diff
@@ -44,7 +44,7 @@ const (
 )
 
 // AllContainers specifies that all containers be visited
-const AllContainers ContainerType = (InitContainers | Containers | EphemeralContainers)
+const AllContainers = InitContainers | Containers | EphemeralContainers
```

Comment:
I think this is a net-negative.  It relies on a subtle facet of Go that even I had to convince myself was correct.  Better to stay explicit, IMO.

What motivated this change?